webapp/src/main/resources/public/js/components/workbench/TranslationHistoryModal.js

```diff
@@ -0,0 +1,122 @@
+                    <td>{item.content}</td>
+                    <td>{this.convertDateTime(item.createdDate)}</td>
+                    <td><StatusGlyph status={item.status} onClick={() => ""}/></td>
+                </tr>
+            ) :
+            "";
+    };
+
+    /**
+     * @returns {*} Generated content for the git blame information section
+     */
+    rendertranslationHistory = () => {
+        const { translationHistory, textUnit, intl } = this.props;
+
+        return (
+            <Table striped bordered hover responsive="sm">
+                <thead>
+                    <th><FormattedMessage id="textUnit.translationHistoryModal.User"/></th>
+                    <th><FormattedMessage id="textUnit.translationHistoryModal.Translation"/></th>
+                    <th><FormattedMessage id="textUnit.translationHistoryModal.Date"/></th>
+                    <th><FormattedMessage id="textUnit.translationHistoryModal.Status"/></th>
+                </thead>
+                <tbody>
+                {(translationHistory && translationHistory.length) ? translationHistory.map(this.renderHistoryItem) : ""}
+                {this.renderHistoryItem({
+                    createdByUser: {
+                        username: "mojito"
+                    },
+                    content: (<span class="history-initial"><FormattedMessage id="textUnit.translationHistoryModal.InitialPush"/></span>),
+                    createdDate: textUnit.getTmTextUnitCreatedDate(),
+                    status: TextUnitSDK.STATUS.TRANSLATION_NEEDED
+                })}
+                </tbody>
+            </Table>
+        );
+    };
+
+    /**
+     * @param date   An integer representing a datetime
+     * @returns {*}  Human readable version of the given datetime
+     *
+     */
+    convertDateTime = (date) => {
+        if (date === null || isNaN(date)) {
+            return null;
+        }
+
+        let options = {
+            year: 'numeric', month: 'numeric', day: 'numeric',
+            hour: 'numeric', minute: 'numeric'
+        };
+        return (this.props.intl.formatDate(date, options));
+    };
+
+    /**
+     * Closes the modal and calls the parent action handler to mark the modal as closed
+     */
+    closeModal = () => {
+        this.props.onCloseModal();
+    };
+
+    render() {
+        const { textUnit, show } = this.props;
+
+        return textUnit ? (
+            <Modal className={"git-blame-modal"} show={show} onHide={this.closeModal}>
+                <Modal.Header closeButton>
+                    <Modal.Title><FormattedMessage id="workbench.translationHistoryModal.title"/></Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <div className={"row"}>
+                        <div className={"col-sm-4"}>
+                            <div>
+                                <FormattedMessage id={"textUnit.translationHistoryModal.locale"} values={{
```

Comment:
Actually, can we merge this into the title since it is almost the same string?

If not merging I'd have a table to align everything, also see comment bellow

Reply:
Done